src/main/java/Duke.java

```diff
@@ -1,10 +1,119 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final Scanner in = new Scanner(System.in);
+
+    /**
+     *
+     * List of all Tasks
+     */
+    private static Task[] allTasks;
+
+    private static String input;
+    private static String logo = getLogo();
```

Comment:
Agree with Zhen Hong as well. Consider storing logo and line as a final String variable. If the logo takes up a lot of space, consider defining a new Java class and putting all these constants in there.